cloudinary_cli/utils/api_utils.py

```diff
@@ -15,6 +15,11 @@
 def query_cld_folder(folder):
     files = {}
     next_cursor = True
+
+    if folder == "/":
```

Comment:
It would be better to handle generic case, when folder ends with `/`( which we might get from the user)

I suggest changing it to:
`folder = folder.rstrip('/')`